### PR TITLE
Edit Check Counts functional tests and refactor of page transitions

### PR DESCRIPTION
**Pull request by porterbot**

### CODE REVIEW

File: test/functional/cucumber/MacroEditCheckCounts.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#https://github.com/cfpb/hmda-pilot/issues/29
+
+@editCheck
+Feature: User can view a loan-level report of the syntactical/validity edits
```
Comment (poorgeek) [inline test/functional/cucumber/MacroEditCheckCounts.feature]: Need to update the feature and scenario to match the filename

File: test/functional/cucumber/QualityEditCheckCounts.feature [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#https://github.com/cfpb/hmda-pilot/issues/29
+
+@editCheck
+Feature: User can view a loan-level report of the syntactical/validity edits
```
Comment (poorgeek) [inline test/functional/cucumber/QualityEditCheckCounts.feature]: Need to update the feature and scenario to match the filename

File: test/functional/cucumber/SyntacticalEditCheckCounts.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#https://github.com/cfpb/hmda-pilot/issues/29
+
+@editCheck
+Feature: User can view a loan-level report of the syntactical/validity edits
```
Comment (poorgeek) [inline test/functional/cucumber/SyntacticalEditCheckCounts.feature]: Need to update the feature and scenario to match the filename

File: test/functional/cucumber/ValidityEditCheckCounts.feature [hunk 4]
```diff
@@ -0,0 +1,54 @@
+#https://github.com/cfpb/hmda-pilot/issues/29
+
+@editCheck
+Feature: User can view a loan-level report of the syntactical/validity edits
```
Comment (poorgeek) [inline test/functional/cucumber/ValidityEditCheckCounts.feature]: Need to update the feature and scenario to match the filename

**Comment by henrygrover** [comment]
Hey David, is MacroEditCheckCounts.feature working for you? For me, it seems to get stuck as a result of any files that have syntactical/validity errors, such as the Q015 file

**Pull request closed without merging**